src/test/java/com/easypost/BetaStatelessRateTest.java

```diff
@@ -0,0 +1,61 @@
+package com.easypost;
```

Comment:
This file needs to be renamed to `BetaRateTest` since all test files must match the lib file and eventually this could contain more than just a stateless rate.